janitor.conf.example

```diff
@@ -1,55 +1,77 @@
-# Database location: must be a valid postgresql URL
+# Database location; must be a valid PostgreSQL URL
 database_location: "postgresql://janitor@example.com:5432/janitor"
 
-# Redis
+# Redis location; must be a valid Redis URL
 redis_location: "redis://localhost"
 
-# Set the default committer; if unset defaults to current users' preferences
-# (i.e. ~/.gitconfig)
+# Default committer to use when committing changes
+# If unset, defaults to current users' preferences (i.e. ~/.gitconfig)
 committer: "Janitor Bot <janitor@jelmer.uk>"
 
-# Origin set in published archives
+# Origin in published archives
 origin: "janitor.debian.net"
 
-# Artifact location: used to store build results. Can be:
-#  + a local file path
+# Store build results (artifacts) location; can be:
```

Comment:
This is less readable than the previous description.

Reply:
Updated